### ISSUE TITLE
Project is broken on Godot 3.2 (Arch Linux)

### ISSUE THREAD

**Issue by PLUkraine** [AntonWedemeier/Godot-FogOfWar#1]
Hi!
I have cloned and imported the project in Godot 3.2. However, I get error when Godot is trying to convert the project to this new version.
![Uploading Screenshot from 2020-04-04 20-42-04.png…]()

**Comment by AntonWedemeier** [comment]
I just tried it myself and opened the project, downloaded from github, with Godot version 3.2.1. It works for me. There are 3 warnings: I am not using the 'delta' parameter; and I don't have icon.png in the res: \\ folder; these have no further influence on the operation.
I couldn't open the added png file, but I'm not an expert so maybe I'm not doing it right.

**Comment by AntonWedemeier** [comment]
Sorry I did not want to close this.

**Comment by AntonWedemeier** [comment]
I solved the 3 warnings.

**Comment by PLUkraine** [comment]
Sorry for the broken screenshot!
On Arch Linux, Godot 3.2.1 I get this error
![Screen](https://i.ibb.co/HHbqdmm/Screenshot-from-2020-04-05-09-04-07.png)
I tried to fix it by manually pointing to the scene. However it doesn't solve the issue

I tried to open the project on a Windows machine and it works fine, no problems there

This seems to be a bug in Godot itself, specifically on my system. Thanks for looking into it!
